wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/consentprompt/ConsentPrompt.kt

```diff
@@ -1,18 +1,24 @@
 package com.android.identity_credential.wallet.ui.destination.consentprompt
 
+//import com.android.identity. .credential.CredentialRequest
+//import com.android.identity.credentialtype.CredentialTypeRepository
```

Comment:
Leftover?

Reply:
gah! thanks for spotting this - I've cleaned it out!